open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -34,7 +34,6 @@
     "* Kernel Density Estimation Plots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            assert test_df["c2"].equals(assert_df["c2"])
</pre><p>c2 should not be changed. Why you encode c1 but c2 gets updated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Finished